pages/glossary/distinguishing-styles.md

```diff
@@ -0,0 +1,15 @@
+--
+title: Distinguishing Styles
+key: distinguishing-styles
+unambiguous: true
+objective: true
+--
+
+Two elements have distinguishing styles if any of the following is true:
+
+- they have [distinguishing borders](#distinguishing-borders); or
+- they have [distinguishing box-shadows](#distinguishing-box-shadows); or
```

Comment:
Seems that borders / box-shadows is addressed by those other two rules.

Reply:
They are, but nevertheless they are distinguishing styles and I would argue for leaving them in this definition (that might be used in other rules).